monkey/infection_monkey/puppet/puppet.py

```diff
@@ -41,11 +41,13 @@ def __init__(
     def load_plugin(self, plugin_type: AgentPluginType, plugin_name: str, plugin: object) -> None:
         self._plugin_registry.load_plugin(plugin_type, plugin_name, plugin)
 
-    def run_credential_collector(self, name: str, options: Dict) -> Sequence[Credentials]:
+    def run_credential_collector(
+        self, name: str, host: TargetHost, options: Dict, interrupt: Event
+    ) -> Sequence[Credentials]:
```

Comment:
Why would the collector need a target host? Credentials collectors run locally.

Reply:
I though that it needed it for the `CredentialsStolenEvent`

Reply:
The target will just be `None` in the event which means it's the local system.